pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -2316,6 +2316,11 @@ public class ServiceConfiguration implements PulsarConfiguration {
         doc = "If true, export topic level metrics otherwise namespace level"
     )
     private boolean exposeTopicLevelMetricsInPrometheus = true;
+    @FieldContext(
+            category = CATEGORY_METRICS,
+            doc = "If true, export buffered metrics"
+    )
+    private boolean enableExposeBufferMetrics = false;
```

Comment:
what about `metricsBufferResponse` ?

the prefix "metrics" makes it clearer that this is something related to metrics
I would drop "enable" as it is not a "feature" but more a optimisation, probably in the next release we will turn this on by default (maybe you can create a issue)